WordPress/Classes/ViewRelated/System/WPSplitViewController.swift

```diff
@@ -14,6 +14,8 @@ import WordPressShared
 
 class WPSplitViewController: UISplitViewController {
 
+    private let quickStartNavigationSettings = QuickStartNavigationSettings()
```

Comment:
In an ideal world, WPSplitViewController wouldn't need to know anything about quick start settings (hence the previous implementation), but as that's not working any more and there's a chance we may revisit WPSplitViewController in the not too distant future I think this is okay for now.